libs/eo/metering-points/shell/src/lib/eo-metering-points-shell.component.ts

```diff
@@ -20,27 +20,59 @@ import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
 
 import { WATT_CARD } from '@energinet-datahub/watt/card';
 import { WattToastService } from '@energinet-datahub/watt/toast';
+import { WattValidationMessageComponent } from '@energinet-datahub/watt/validation-message';
 
 import { EoMeteringPointsTableComponent } from './eo-metering-point-table.component';
 import { EoMeteringPointsStore } from './eo-metering-points.store';
-import { EoBetaMessageComponent } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
-  imports: [AsyncPipe, NgIf, EoMeteringPointsTableComponent, WATT_CARD, EoBetaMessageComponent],
+  imports: [
+    AsyncPipe,
+    NgIf,
+    EoMeteringPointsTableComponent,
+    WATT_CARD,
+    WattValidationMessageComponent,
+  ],
   selector: 'eo-metering-points-shell',
   styles: [
     `
-      watt-card-title {
+      :host {
         display: flex;
-        gap: var(--watt-space-m);
-        align-items: center;
+        flex-direction: column;
+        gap: var(--watt-space-l);
       }
     `,
   ],
   template: `
-    <eo-eo-beta-message></eo-eo-beta-message>
+    <watt-validation-message
+      label="You have the option to switch your metering points ON and OFF."
+      size="normal"
+      icon="info"
+      [autoScrollIntoView]="false"
+    >
+      <p>For Production metering points:</p>
+
+      <ul>
+        <li>Turning it 'ON' means the metering point is actively issuing certificates of power.</li>
```

Comment:
Hvor eksplicitte skal vi være? skal vi gå ud fra at brugere forstår at det kun er produktions målepunkter, fordi de befinder sig på produktions målepunkt siden, eller skal vi være helt eksplicitte, og skrive "Turning it 'ON' means the metering point is actively issuing **_production_** certificates of power.